features/webgl-compressed-texture-pvrtc.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-07-11
-name: WebGL WEBGL_compressed_texture_pvrtc Extension Specification
-description: TODO
+name: WebGL PVRTC texture support
+description: (DEPRECATED) Exposes the compressed PVRTC texture formats.
```

Comment:
This is marked as *deprecated* in the spec, not sure how we're supposed to handle it here. It's got pretty limited support.

```
status:
  baseline: false
  support:
    chrome_android: "28"
    safari: "7"
    safari_ios: "7"
```